package.json

```diff
@@ -15,15 +15,17 @@
     "@fontsource/roboto": "^5.0.13",
     "@fortawesome/fontawesome-free": "^6.5.2",
     "@mui/icons-material": "^5.15.18",
-    "@mui/material": "^5.15.18",
+    "@mui/lab": "^5.0.0-alpha.170",
+    "@mui/material": "^5.15.19",
     "axios": "^1.7.2",
     "jwt-decode": "^4.0.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.51.5",
     "react-pro-sidebar": "^1.1.0",
     "react-router-dom": "^6.23.1",
-    "react-toastify": "^10.0.5"
+    "react-toastify": "^10.0.5",
```

Comment:
We might look for the snackbar component from MUI
https://mui.com/material-ui/react-snackbar/